### PR TITLE
Speedup user list page loading time

### PR DESCRIPTION
**Pull request by ender503**
By moving the API called from server-side into client-side, the user
list page should be loaded fast.

### CODE REVIEW

File: pages/users.js [hunk 1]
```diff
@@ -1,46 +1,45 @@
...
+    this.state = {
+      userList: [],
+    };
+  }
 
-UsersPage.propTypes = {
-  userList: PropTypes.array,
-};
+  componentDidMount() {
+    fetch('/api/fetchUserList')
```
Comment (jserv) [inline pages/users.js]: How can we extend routing path? We should have an explicit list of `/api/` which points to actual response.

**Comment by ender503** [comment]
Add the `tangleidAPI.js` utils to manage the Tangle ID API calls. :+1;

**Comment by ender503** [comment]
I describe the reason of slow down in the commit message. Thanks for your reminder.

**Comment by jserv** [comment]
Append the issue number you are about to close in Git commit message.

**Comment by ender503** [comment]
This PR is related to issue #38, but this PR only fixed the user list page.
Other pages still have the same problem, I will fix theme in the following PR.

**Comment by jserv** [comment]
s/following PRs/incoming PR/

**Pull request merged**